fabric-networking-api-v1/src/main/java/net/fabricmc/fabric/mixin/networking/ClientConnectionMixin.java

```diff
@@ -109,17 +106,8 @@ private void disconnectAddon(CallbackInfo ci) {
 		}
 	}
 
-	@Inject(method = "sendInternal", at = @At(value = "INVOKE", target = "Lio/netty/channel/ChannelFuture;addListener(Lio/netty/util/concurrent/GenericFutureListener;)Lio/netty/channel/ChannelFuture;", remap = false), locals = LocalCapture.CAPTURE_FAILHARD, cancellable = true)
-	private void sendInternal(Packet<?> packet, @Nullable PacketCallbacks callbacks, boolean flush, CallbackInfo ci, ChannelFuture channelFuture) {
-		if (callbacks instanceof GenericFutureListenerHolder holder) {
-			channelFuture.addListener(holder.getDelegate());
-			channelFuture.addListener(ChannelFutureListener.FIRE_EXCEPTION_ON_FAILURE);
-			ci.cancel();
-		}
-	}
-
 	@Override
-	public Collection<Identifier> getPendingChannelsNames(NetworkState state) {
+	public Collection<Identifier> getPendingChannelsNames(NetworkPhase state) {
```

Comment:
Needs prefixing? Or is it too late

Reply:
Done, this is impl so can be renamed freely.